recipes/ldas-tools-frameapi-swig/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "ldas-tools-frameAPI-swig" %}
+{% set version = "2.6.7" %}
+
+# dependencies
+{% set ldas_tools_cmake_version = "1.0.5" %}
+{% set ldas_tools_frameapi_version = "2.6.3" %}
+{% set ldas_tools_framecpp_swig_version = "2.6.6" %}
+{% set ldas_tools_ldasgen_swig_version = "2.6.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz"
+  sha256: b44409d16ca35e648ff3aa05866eb1f679049606fd6b80f451f24fc24fdcaa5e
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.6
+    - pkg-config  # [not win]
+    - make  # [not win]
+    - swig 3
+  host:
+    - ldas-tools-cmake >={{ ldas_tools_cmake_version }}
+    - ldas-tools-frameapi >={{ ldas_tools_frameapi_version }}
+    - ldas-tools-framecpp-swig >={{ ldas_tools_framecpp_swig_version }}
+    - ldas-tools-ldasgen-swig >={{ ldas_tools_ldasgen_swig_version }}
+  run:
+    - ldas-tools-frameapi >={{ ldas_tools_frameapi_version }}
+    - ldas-tools-framecpp-swig >={{ ldas_tools_framecpp_swig_version }}
+    - ldas-tools-ldasgen-swig >={{ ldas_tools_ldasgen_swig_version }}
```

Comment:
I think this output also needs the `about` section

Reply:
@scopatz, I'm confused, there is a top-level `about` section that should be used by the `ldas-tools-frameapi-swig` `output` because it has the same name as the recipe. If I run `conda build ldas-tools-frameapi-swig` I get a package for that output that seems to have everything populated in the `info/` dir and the rendered `meta.yaml` as expected. This follows the same pattern [used by matplotlib](https://github.com/conda-forge/matplotlib-feedstock/blob/5af510232c2fa984e981e3eeb18128a6cab41b6d/recipe/meta.yaml).

Reply:
<details>
<summary>Conda render output for ldas-tools-frameapi-swig</summary>

```
----------
meta.yaml:
----------
package:
    name: ldas-tools-frameapi-swig
    version: 2.6.7
source:
    sha256: b44409d16ca35e648ff3aa05866eb1f679049606fd6b80f451f24fc24fdcaa5e
    url: http://software.ligo.org/lscsoft/source/ldas-tools-frameAPI-swig-2.6.7.tar.gz
build:
    number: '0'
requirements:
    build:
        - zlib 1.2.11 h01d97ff_1006
        - make 4.2.1 h1de35cc_2004
        - clang_osx-64 9.0.0 h22b1bf0_0
        - bzip2 1.0.8 h01d97ff_1
        - rhash 1.3.6 h1de35cc_1001
        - xz 5.2.4 h1de35cc_1001
        - tk 8.6.9 h2573ce8_1003
        - clang 9.0.0 h28b9765_0
        - cmake 3.15.3 hdd2e4aa_0
        - curl 7.65.3 h22ea746_0
        - clangxx_osx-64 9.0.0 h22b1bf0_0
        - cctools 921 h5ba7a2e_2
        - ca-certificates 2019.9.11 hecc5488_0
        - libcxx 9.0.0 0
        - ncurses 6.1 h0a44026_1002
        - libcxxabi 9.0.0 0
        - libedit 3.1.20170329 hcfe32e1_1001
        - libcurl 7.65.3 h16faf7d_0
        - tapi 1000.10.8 h770b8ee_1
        - openssl 1.1.1c h01d97ff_0
        - pcre 8.41 h0a44026_1003
        - libssh2 1.8.2 hcdc9a53_2
        - krb5 1.16.3 hcfa6398_1001
        - swig 3.0.12 h0a44026_1003
        - libiconv 1.15 h01d97ff_1005
        - pkg-config 0.29.2 h112b980_1005
        - compiler-rt 9.0.0 hce3ea14_0
        - libllvm9 9.0.0 h770b8ee_2
        - clangxx 9.0.0 0
        - expat 2.2.5 h6de7cb9_1003
        - ld64 409.12 h3c32e8a_2
        - libuv 1.31.0 h01d97ff_0
    host:
        - bzip2 1.0.8 h01d97ff_1
        - ca-certificates 2019.9.11 hecc5488_0
        - libcxxabi 9.0.0 0
        - xz 5.2.4 h1de35cc_1001
        - zlib 1.2.11 h01d97ff_1006
        - libcxx 9.0.0 0
        - openssl 1.1.1c h01d97ff_0
        - icu 64.2 h6de7cb9_1
        - ldas-tools-al 2.6.2 he3f880d_1003
        - ldas-tools-cmake 1.0.7 h0a44026_0
        - boost-cpp 1.70.0 h75728bb_2
        - ldas-tools-filters 2.6.3 h04f5b5a_1000
        - ldas-tools-ldasgen 2.6.3 hf93a975_1003
        - ldas-tools-framecpp 2.6.5 he3f880d_1004
        - ldas-tools-ldasgen-swig 2.6.6 h04f5b5a_1000
        - ldas-tools-frameapi 2.6.3 h153e6df_1001
        - ldas-tools-framecpp-swig 2.6.8 h0a44026_0
    run:
        - ldas-tools-framecpp-swig >=2.6.6
        - libcxx >=9.0.0
        - ldas-tools-frameapi >=2.6.3
        - ldas-tools-ldasgen-swig >=2.6.6
test:
    commands:
        - test -f ${PREFIX}/include/frameAPI/LDASframe.i
outputs:
    -   name: ldas-tools-frameapi-swig
    -   about:
            dev_url: https://git.ligo.org/ldastools/LDAS_Tools
            home: https://wiki.ligo.org/Computing/DASWG/LDASTools
            license: GPLv3
            license_family: GPL
            license_file: COPYING
            summary: Python bindings for the LDAS Tools frame utility library
        build:
            activate_in_script: true
        name: python-ldas-tools-frameapi
        requirements:
            build:
                - clangxx_osx-64
                - cmake >=3.6
                - pkg-config
                - make
                - swig 3
            host:
                - ldas-tools-cmake >=1.0.5
                - ldas-tools-al
                - ldas-tools-al-swig
                - ldas-tools-frameapi-swig 2.6.7 0
                - python
            run:
                - python
                - ldas-tools-al
                - ldas-tools-frameapi >=2.6.3
                - ldas-tools-frameapi-swig 2.6.7 0
                - python-ldas-tools-framecpp >=2.6.6
                - python-ldas-tools-ldasgen >=2.6.6
        script: install-python.sh
        test:
            imports:
                - LDAStools.LDASlogging
about:
    dev_url: https://git.ligo.org/ldastools/LDAS_Tools
    home: https://wiki.ligo.org/Computing/LDASTools
    license: GPLv2+
    license_family: GPL
    license_file: COPYING
    summary: SWIG interface for the LDAS Tools frame utility library
extra:
    copy_test_source_files: true
    final: true
    recipe-maintainers:
        - duncanmmacleod
        - emaros
```
</details>

Reply:
Ahh yeah, sorry I was reviewing this on my phone and did not do a good job